Ubiquity Journals.js

```diff
@@ -0,0 +1,156 @@
+{
+	"translatorID": "5c11e3bd-caf5-4da6-95d8-e67c57929098",
+	"label": "Ubiquity Journals",
+	"creator": "Sebastian Karcher",
+	"target": "/articles/10\\.\\d{4,9}/[-._;()/:a-z0-9A-Z]+|/articles/?$|/\\d+/volume/\\d+/issue/\\d+",
```

Comment:
Should we force here also that there is no other substructere before, i.e. something like
```
"target": "^[^/]+(/articles/10\\.\\d{4,9}/[-._;()/:a-z0-9A-Z]+|/articles/?$|/\\d+/volume/\\d+/issue/\\d+)",
```
?